README.md

```diff
@@ -202,6 +204,17 @@ Phantoms will no longer spawn if the hostile mobcap is full. This is per player.
 - Required options: `true`, `false`
 - Categories: `FEATURE`, `CARPET-ADDONS-NOT-FOUND`
 
+### portalCreativeDelay
+
+Amount of delay ticks to use a nether portal in creative
+
+- Type: `Integer`
+- Default value: `1`
+- Suggested options: `1`, `40`, `80`, `72000`
+- Categories: `CREATIVE`, `CARPET-ADDONS-NOT-FOUND`
+- Additional notes:
```

Comment:
We should have a note saying that this rule will only be considered when the `spectatorPlayersUsePortals` rule is enabled.

I’ve made a comment on the description of the rule in `en_us.json`. Please take a look at that before addressing this one.